mandatory/2-function-creation.js

```diff
@@ -16,16 +18,34 @@ Write a function that:
 - numbers greater 100 must be replaced with 100
 */
 
-function formatPercentage(num) {}
+function formatPercentage(num) {
+  if (num > 100) num = 100;
+  let roundedNum =  Math.round(num * 100) / 100;// this is comparing  
+  return `${roundedNum}%`; 
+}
 
+
+function formatPenceToPounds(price) {
+  return '£' +( price / 100).toFixed(2); //this will create 2 decimal in the price
+}
+
+// let formattedPrice = formatPenceToPounds(199);
 /*
 Write a function that:
 - takes an array of strings as input
 - removes any spaces in the beginning or end of each string
 - removes any forward slashes (/) in each string
 - makes all strings all lowercase
 */
-function tidyUpStrings(arrayOfStrings) {}
+function tidyUpStrings(arrayOfStrings) {
+  let newArr = [];
+  for (const element of arrayOfStrings) {
```

Comment:
good solution. you can use map method as well.

Reply:
thank you Leila ❤️